cognite/neat/rules/exporter/rules2excel.py

```diff
@@ -0,0 +1,159 @@
+        # Serialize the rules to the excel file
+
+        # map rules metadata to excel
+        metadata = self.rules.metadata
+        metadata_sheet = self.workbook.create_sheet("Metadata")
+
+        # add each metadata property to the sheet as a row
+
+        metadata_sheet.append(["title", metadata.title])
+        metadata_sheet.append(["description", metadata.description])
+        metadata_sheet.append(["version", metadata.version])
+        metadata_sheet.append(["creator", ",".join(metadata.creator)])
+        metadata_sheet.append(["created", metadata.created])
+        metadata_sheet.append(["dataModelName", metadata.data_model_name])
+        metadata_sheet.append(["cdfSpaceName", metadata.cdf_space_name])
+
+        metadata_sheet.append(["prefix", metadata.prefix])
+        metadata_sheet.append(["namespace", metadata.namespace])
+
+        # map classes to excel sheet named "Classes" and add each class as a row
+        classes_sheet = self.workbook.create_sheet("Classes")
+
+        classes_sheet.append(["Solution model", "", ""])
+        classes_sheet.merge_cells("A1:C1")
+        classes_sheet.append(["Class", "Description", "Parent Class"])  # A  # B  # C
+
+        for class_ in self.rules.classes.values():
+            self.class_counter += 1
+            classes_sheet.append([class_.class_id, class_.description, class_.parent_class])
+
+        # map properties to excel sheet named "Properties" and add each property as a row
+        properties_sheet = self.workbook.create_sheet("Properties")
+        properties_sheet.append(
+            [
+                "Solution model",  # A
+                "",  # B
+                "",  # C
+                "",  # D
+                "",  # E
+                "",  # F
+                "Solution classic CDF resources",  # G
+                "",  # H
+                "",  # I
+                "",  # J
+                "",  # K
+                "",  # L
+                "Transformation rules",  # M
+                "",  # N
+            ]
+        )
+        properties_sheet.merge_cells("A1:F1")
+        properties_sheet.merge_cells("G1:L1")
+        properties_sheet.merge_cells("M1:N1")
+        properties_sheet.append(
+            [
+                "Class",  # A
+                "Property",  # B
+                "Description",  # C
+                "Type",  # D
+                "Min Count",  # E
+                "Max Count",  # F
+                "Resource Type",  # G
+                "Resource Type Property",  # H
+                "Relationship Source Type",  # I
+                "Relationship Target Type",  # J
+                "Relationship Label",  # K
+                "Relationship ExternalID Rule",  # L
+                "Rule Type",  # M
+                "Rule",  # N
+            ]
+        )
+
+        for property_ in self.rules.properties.values():
+            self.property_counter += 1
+            properties_sheet.append(
```

Comment:
How does it handle data modeling only TRules, i.e. TRules without rules?

Reply:
It just serializes TRules object into excel. Rules is just a column , I guess it should work just fine. I think @nikokaoja can bring many improvements here.